log-output/pkg/index/service.go

```diff
@@ -78,9 +106,14 @@ func (s Service) ListIndices(w http.ResponseWriter, r *http.Request) {
 
 	s.m.RUnlock()
 
-	_template, err := template.New("listPage").Funcs(template.FuncMap{"DateFormat": DateFormat}).Parse(listPageTemplate)
-	_ = err
-	_ = _template.Execute(w, indices)
-	w.Header().Add("Content-Type", "text/html")
-}
+	if r.Header.Get("Accept") == "application/json" {
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(indices)
```

Comment:
nit: can you add `_ =`, I like to call out that we are explicitly ignoring an error.